### PR TITLE
Update master azure pipeline yaml file

### LINKED ISSUE #68: Feature/Setup Pipeline in Azure Devops
## Description
As a developer, I want to integrate unit testing in the pipeline. 

## Definitions of Done
- [x] Perform unit test
- [x] Passes Build
- [ ] Generate artifact
- [x] Code coverage report

### PR DESCRIPTION
**Pull request by kimlisa**
closes #68

### CODE REVIEW

File: azure-pipelines-pr-template.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Runs for every pull request on master
```
Comment (faraonc) [inline azure-pipelines-pr-template.yml]: rename this file to `azure-pipelines-pr-test.yml`

File: azure-pipelines-pr-template.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      export hosts_postgres_host="localhost"
+      export hosts_postgres_db="test_user_svc"
+      export hosts_postgres_user="postgres"
+      export hosts_postgres_password="secret"
+      export hosts_postgres_sslmode="disable"
+      export hosts_postgres_port="5432"
+      export hosts_smtp_host="smtp.gmail.com"
+      export hosts_smtp_port="587"
+      export hosts_smtp_username="hwsc.test@gmail.com"
+      export hosts_smtp_password="ftbsflgbewdpzzmo"
```
Comment (faraonc) [inline azure-pipelines-pr-template.yml]: is this okay to be exposed?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -23,24 +26,89 @@ pool:
...
+      export hosts_postgres_host="localhost"
+      export hosts_postgres_db="test_user_svc"
+      export hosts_postgres_user="postgres"
+      export hosts_postgres_password="secret"
+      export hosts_postgres_sslmode="disable"
+      export hosts_postgres_port="5432"
+      export hosts_smtp_host="smtp.gmail.com"
+      export hosts_smtp_port="587"
+      export hosts_smtp_username="hwsc.test@gmail.com"
+      export hosts_smtp_password="ftbsflgbewdpzzmo"
```
Comment (faraonc) [inline azure-pipelines.yml]: Make this is as an environment variable in in DevOps pipeline if this is not safe.

File: azure-pipelines-pr-template.yml (hunk 1 above)
Comment (kimlisa) [reply azure-pipelines-pr-template.yml]: i'll name it to that when i build another pipeline in master with designer. this is just a temporary file

**Pull request merged**